04-store-api/starter/app.js

```diff
@@ -1 +1,45 @@
-console.log('04 Store API')
+require('dotenv').config();
```

Comment:
A comment explaining why you're doing a global-style import here is useful. While programming, think about if you're in a situation where the "why" of your code isn't immediately clear. That's the gap that good code comments fill.

You could even just put a link to the docs do a good job of explaining what's going on here: https://www.npmjs.com/package/express-async-errors#how-does-this-work